examples/resources/hpegl_metal_volume/resource.tf

```diff
@@ -7,16 +7,16 @@ provider "hpegl" {
 }
 
 variable "location" {
-  default = "USA:Central:AFCDCC1"
+  default = "USA:CO:FTC"
```

Comment:
as I recall, this file is setup to work well with simulated resources, why this change?